src/components/forms/RegisterForm.js

```diff
@@ -40,36 +37,53 @@ class RegisterForm extends Component {
     const { username, email, password } = this.state;
 
     return (
-      <div>
-        <form onClick={this.handleClick} onSubmit={this.onFormSubmit}>
-          <label htmlFor="username">Username:</label>
-          <input
-            type="text"
-            value={username}
-            autoComplete="username"
-            onChange={e => this.onInputChange("username", e)}
-          />
+      <Container>
```

Comment:
Register function remain the same. I just styled the form here!

Reply:
cool